samples/peripherals/pwm/README.rst

```diff
@@ -0,0 +1,96 @@
+.. _pwm_sample:
+
+PWM
+###
+
+.. contents::
+   :local:
+   :depth: 2
+
+The PWM sample demonstrates how to configure and use PWM with |BMlong|.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. tabs::
+
+   .. group-tab:: Simple board variants
+
+      The following board variants do **not** have DFU capabilities.
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Hardware platform
+           - PCA
+           - SoftDevice
+           - Board target
+         * - `nRF54L15 DK`_
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice
+         * - `nRF54L15 DK`_ (emulating nRF54L10)
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice
+         * - `nRF54L15 DK`_ (emulating nRF54L05)
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice
+
+   .. group-tab:: MCUboot board variants
+
+      The following board variants have DFU capabilities.
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Hardware platform
+           - PCA
+           - SoftDevice
+           - Board target
+         * - `nRF54L15 DK`_
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice/mcuboot
+         * - `nRF54L15 DK`_ (emulating nRF54L10)
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice/mcuboot
+         * - `nRF54L15 DK`_ (emulating nRF54L05)
+           - PCA10156
+           - S115
+           - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice/mcuboot
+
+Overview
+********
+
+The sample initializes a PWM instance and use that to iluminate LED1 and LED3 on the device.
+
+.. note::
+
+  PWM signal can be exposed on GPIO pin only within same domain.
+  For nRF54L-series there is only one domain which contains both PWM and GPIO: PWM20/21/22 and GPIO Port P1.
+  Only LEDs connected to P1 can work with PWM, in this case LED1 and LED3.
```

Comment:
I think it could be a good idea to insert here a link to the datasheet or some reference where the user can check this information.